.travis.yml

```diff
@@ -11,5 +11,5 @@ before_install:
   - tar xfz mongodb-linux-x86_64-$MONGODB_VERSION.tgz
   - export PATH=`pwd`/mongodb-linux-x86_64-$MONGODB_VERSION/bin:$PATH
   - mkdir -p data/db
-  - mongod --dbpath=data/db &
+  - mongod --dbpath=data/db --setParameter ttlMonitorSleepSecs=2 &
```

Comment:
why is this parameter needed?

Reply:
Mongodb ttl background task by default runs every 60 seconds (https://docs.mongodb.com/manual/core/index-ttl/#timing-of-the-delete-operation), that is why last test that checks if a retained packet is automatically deleted when setting a ttl takes 65 seconds making all tests so long. This speeds up that task and makes it run every 2 seconds

Reply:
I have set it also from the test ci here: https://github.com/mcollina/aedes-persistence-mongodb/pull/29/files/73142765694972e2587c63786bf4faa29ff481fb#diff-1dd241c4cd3fd1dd89c570cee98b79ddR23

But seems that in travis that is not picked up. In my local install it works even without starting mongo with this parameter